src/NearbyInteraction/NINearbyObject.cs

```diff
@@ -41,7 +43,22 @@ partial class NINearbyObject
 				return (Vector3)_DirectionNotAvailable;
 			}
 		}
-	}
 
+		static MatrixFloat4x4? _WorldTransformNotAvailable;
+
+		// Following similar strategy found here: https://github.com/xamarin/maccore/issues/2274
+		[Field ("NINearbyObjectWorldTransformNotAvailable",  "NearbyInteraction")]
+		public static MatrixFloat4x4 WorldTransformNotAvailable {
+			get {
+				if (_WorldTransformNotAvailable is null) {
+					unsafe {
+						MatrixFloat4x4 *pointer = (MatrixFloat4x4 *) Dlfcn.GetIndirect (Libraries.NearbyInteraction.Handle, "NINearbyObjectWorldTransformNotAvailable");
+						_WorldTransformNotAvailable = *pointer;
```

Comment:
I think this will crash if executed on an earlier OS version (where the native field doesn't exist).

Something like this should work instead:

```suggestion
						if (pointer == null)
							throw new PlatformNotSupportedException ("This property is not supported on this version of the OS");
						_WorldTransformNotAvailable = *pointer;
```

The property also needs availability attributes.